app/views/admin/timelogs/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="well">
+  <% if params[:timelogs] == 'false' %>
```

Comment:
this is weird, do you have `params[:timelogs]`? You should check based on `@timelogs`